### ISSUE TITLE
update with following error:

### ISSUE THREAD

**Issue by soki3456** [nextcloud/server#8431]
Cannot login, because the update fails. 

--> 
### Steps to reproduce
1. updater manager - new version 13.0.0
2. clicking through
3.

### Expected behaviour
installation of the update

### Actual behaviour
**Doctrine\DBAL\Exception\TableExistsException: An exception occurred while executing 'ALTER TABLE oc_jobs CHANGE `id` id BIGINT UNSIGNED AUTO_INCREMENT NOT NULL': SQLSTATE[42S01]: Base table or view already exists: 1050 Table './streifzuege_wolke/#sql2-30f-607449' already exists
Detaillierte Protokollmeldungen
Update vorbereiten
Log-Level auf "debug" gesetzt
Wartungsmodus eingeschaltet
Das Datenbankschema wird aktualisiert
Doctrine\DBAL\Exception\TableExistsException: An exception occurred while executing 'ALTER TABLE oc_jobs CHANGE `id` id BIGINT UNSIGNED AUTO_INCREMENT NOT NULL': SQLSTATE[42S01]: Base table or view already exists: 1050 Table './streifzuege_wolke/#sql2-30f-607449' already exists
Das Update ist fehlgeschlagen. Bitte melde dieses Problem an die Nextcloud Community.**

### Server configuration

**Operating system**: sorry don't knwo 

**Web server:** 

**Database:** mySQL

**PHP version:** 7.0

**Nextcloud version:** (see Nextcloud admin page) - previous to 13.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:** update

**Where did you install Nextcloud from:** 

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by soki3456** [comment]
https://wolke.streifzuege.org

**Comment by MorrisJobke** [comment]
Looks like #8912 is something similar but for the `oc_comments` table.

@blizzz @icewind1991 @nickvergessen Have you ever seen this and an idea what this could be?

**Comment by nickvergessen** [comment]
not really

**Comment by octogene** [comment]
Resolved #8912 by doing a FLUSH TABLE on `oc_comments`. Had the same kind of error on `oc_calendarobjects` after re-running `./occ upgrade` but after doing a FLUSH TABLE on `oc_calendarobjects` the upgrade finished without error.
